packages/components/organisms/f-header/src/components/Header.vue

```diff
@@ -144,6 +143,9 @@ export default {
 
         headerBackgroundClass () {
             return this.headerBackgroundTheme === 'highlight' ? 'c-header--highlightBg' : '';
+        },
+        showOffersLinkWithContent () {
```

Comment:
For consistency with the rest of the computed properties here:
```suggestion

        showOffersLinkWithContent () {
```